File: client/bcephal-client-model/Billing/Model/BillingModelEditorData.cs
using Bcephal.Models.Base;
using Bcephal.Models.Grids;
using System;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Bcephal.Models.Billing.Model
{
    public class BillingModelEditorData : EditorData<BillingModel>
    {

        public ObservableCollection<string> Variables { get; set; }
        public ObservableCollection<string> Locales { get; set; }
        public ObservableCollection<Nameable> Sequences { get; set; }

        public ObservableCollection<GrilleColumn> RepositoryColumns { get; set; }

    }
}

File: server/bcephal-billing/src/main/java/com/moriset/bcephal/billing/domain/BillingModelEditorData.java
/**
 * 
 */
package com.moriset.bcephal.billing.domain;

import java.util.List;

import com.moriset.bcephal.domain.EditorData;
import com.moriset.bcephal.domain.Nameable;

/**
 * @author Joseph Wambo
 *
 */
public class BillingModelEditorData extends EditorData<BillingModel> {

	public List<String> variables;
	
	public List<String> locales;
	
	public List<Nameable> sequences;
	
	public List<Nameable> companies;
	
	public BillingModelEditorData() {
		super();
	}
	
	public BillingModelEditorData(EditorData<BillingModel> data) {
		super(data);
	}
	
}
